server/gokbg3/grails-app/domain/org/gokb/cred/ComponentPrice.groovy

```diff
@@ -16,21 +16,56 @@ class ComponentPrice {
   Float price
 
   static mapping = {
-        owner column:'cp_owner_component_fk'
-    priceType column:'cp_type_fk'
-     currency column:'cp_currency_fk'
-    startDate column:'cp_start_date'
-      endDate column:'cp_end_date'
-        price column:'cp_price'
+    owner column: 'cp_owner_component_fk'
```

Comment:
Is this reformatting done by intention?

Reply:
The reformatting occured as a result of the age of the file. Apparently, it has been untouched for such a long time that slight changes in the coding style over the past years have not affected this (and others).